opencti-platform/opencti-front/src/private/components/settings/metrics/metrics.d.ts

```diff
@@ -0,0 +1,3 @@
+export type metricsGraphqlQueryUser = {
```

Comment:
```suggestion
export type MetricsGraphqlQueryUser = {
```
To keep consistency, types and interfaces start with a capitalise letter